tests/infrastructure/test_kubernetes_job.py

```diff
@@ -924,7 +924,7 @@ def mock_log_stream(*args, **kwargs):
                 func=mock_k8s_batch_client.list_namespaced_job,
                 field_selector=mock.ANY,
                 namespace=mock.ANY,
-                timeout_seconds=pytest.approx(0.5, abs=0.02),
+                timeout_seconds=1,  # ceil(1 - 0.25 - test execution time) should be 1
```

Comment:
This means we are no longer actually testing what this test cares about, we need to make sure that initial sleep time is subtracted